google-api-client/google-api-client.gemspec

```diff
@@ -13,9 +13,16 @@ Gem::Specification.new do |gem|
   gem.metadata      = {
     "documentation_uri" => "https://googleapis.dev/ruby/google-api-client/v#{Google::Apis::VERSION}"
   }
+
+  gem.post_install_message = <<~MESSAGE
+    ** The google-api-client gem is deprecated and will not be updated further. **
```

Comment:
This statement is different from the OVERVIEW, which states:

> This library is **deprecated** and will likely not be updated in the future.

It seems to me that the message should be identical.

Reply:
done